### ISSUE TITLE
Path in state 1 to 3 should transform into grass path when a shovel is used

### ISSUE THREAD

**Issue by NebelNidas** [lebonq/Automatic-Path#24]
Similar to #8, you should be able to transform paths with a decent amount of grass (state 1 to 3) into vanilla grass paths: https://minecraft.fandom.com/wiki/Grass_Path

**Comment by lebonq** [comment]
Thanks for reporting this !

**Comment by tennox** [comment]
Could there be a config option to always make this possible?
In vanilla minecraft you can also make path from dirt, not only grass - it's kinda annoying currency I have to remove and re-place auto-path blocks

**Comment by lebonq** [comment]
ATM there is an option to transform path automatically to vanilla path block.

**Comment by tennox** [comment]
Yes, but as far as I understand only for grass with sufficient grass level - but this has changed even in vanilla to also work on dirt blocks:
https://minecraft.fandom.com/wiki/Dirt_Path#History

I would love it if this mod would also work that way - to allow changing all types of path states to a Dirt Path block with a shovel
